spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/config/GcpConfigBootstrapConfiguration.java

```diff
@@ -35,9 +35,9 @@
  * @author João André Martins
  */
 @Configuration
-@AutoConfigureAfter(GcpContextAutoConfiguration.class)
+@Import(GcpContextAutoConfiguration.class)
 @EnableConfigurationProperties(GcpConfigProperties.class)
-public class GcpConfigAutoConfiguration {
+public class GcpConfigBootstrapConfiguration {
 
 	@Bean
 	@ConditionalOnProperty(prefix = "spring.cloud.gcp.config", name = "enabled", havingValue = "true")
```

Comment:
Isn't `@ConditionalOnProperty` also an autoconfiguration feature? Will it still work for bootstrap configuration?